packages/devtools_app/test/flutter/debugger_screen_test.dart

```diff
@@ -46,6 +46,7 @@ void main() {
       when(debuggerController.breakpoints).thenReturn(ValueNotifier([]));
       when(debuggerController.scriptList)
           .thenReturn(ValueNotifier(ScriptList(scripts: [])));
+      when(debuggerController.sortedScripts).thenReturn(ValueNotifier([]));
```

Comment:
can we add a test where the sortedScripts list is not empty?

Reply:
done!